### DISCUSSION TITLE
osu!mania grades should be based on score, not accuracy

### DISCUSSION THREAD

**Discussion by KcHecKa** [ppy/osu#17850] [Ideas]
this has been brought up by peppy before: https://github.com/ppy/osu/discussions/14757#discussioncomment-1332529

since "greats" in lazer affect your accuracy, rank grades should be based specifically on scores. and I don't think 1,000,000 score should net a different rank, that would just cause confusing, unless we want to add an additional rank above SS, _SSS or Silver SS?_

Here's a broad suggestion of what it could look like: 
- **SS: 950,000**
-  **S: 800,000**
-  **A: 700,000**
-  **B: 600,000**
-  **C: 500,000**
-  **D: <500,000**

can be adjusted depending on how you guys feel

**Comment by frenzibyte** [comment]
The purpose of SS is to indicate perfection, therefore it should probably remain at 1,000,000 rather than being tolerant and only for mania.

**Reply by smoogipoo** [reply]
You can say this but also have to consider that 1M score is extremely rare in mania beyond 2* maps. It's less achievable in practice because PERFECT is a shorter hit window than the GREAT which osu! is capped at.
